contracts/external/proxies/SignedOperationProxy.sol

```diff
@@ -70,7 +70,8 @@ contract SignedOperationProxy is
         "Action[] actions,",
         "uint256 expiration,",
         "uint256 salt,",
-        "address sender",
+        "address sender,",
+        "address signer",
```

Comment:
why do we need this change again? And do we not need to change what we're actually hashing? (I'm not the most familiar with the 712 stuff)

Reply:
This was added to the struct between our 1.0 deploy and now. Now that ganache supports signing arrays with EIP712, I added a test with signing an operation using the `eth_signTypedData` RPC call (aka EIP712). Due to this, I realized that there was a bug with the signatures in solidity/javascript which weren't matching the ganache signature anymore since I did not make those changes to calculating the operationHash now that the signer is included in the operation header.

Reply:
Change to the struct that introduced the problem: https://github.com/dydxprotocol/solo/pull/297/files#diff-8104481c278ed60013172173fa1ec998R136